frontend/index.js

```diff
@@ -0,0 +1,108 @@
+document.addEventListener("DOMContentLoaded", () => {
+   animalBtn();
+   numberBtn();
+   peekBtn();
+   enqueueBtn();
+   dequeueBtn();
+})
+
+let url = "http://localhost:8000/queue"
+
+const animalBtn = () => {
```

Comment:
This is nice and clean to read, but you dont need to make all of this into functions